### PR TITLE
PVP newnitions

### PR DESCRIPTION
**Pull request by Kmc2000**
## About The Pull Request

This PR provides no guarantee that any of the code actually works.

## Why It's Good For The Game

* PVP is fun
* Aetherwhisp is a cool map
* Tuning fork is better
* The game!

## Changelog
:cl: Kmc2000, SergeiKoralev, Aerwhisp, Altangy
add: Added the Aetherwhisp as a PVP ship.
add: Syndicate now have an ID mod console. Added a framework for secondary ship access consoles.
/:cl:

### CODE REVIEW

File: nsv13/code/game/gamemodes/pvp/pvp.dm [hunk 1]
```diff
@@ -13,8 +13,8 @@ GLOBAL_LIST_EMPTY(syndi_crew_leader_spawns)
 	config_tag = "pvp"
 	report_type = "pvp"
 	false_report_weight = 10
-	required_players = 40//40 // 40 to make 20 v 20
-	required_enemies = 14//20
+	required_players = 0//40 // 40 to make 20 v 20
```
Comment (Jalleo) [inline nsv13/code/game/gamemodes/pvp/pvp.dm]: Fix this

File: nsv13/code/modules/mining/asteroid.dm [hunk 2]
```diff
@@ -160,6 +160,11 @@ GLOBAL_LIST_EMPTY(asteroid_spawn_markers)		//handles mining asteroids, kind of s
 	var/turf/target_location = null //Where to load the asteroid
 	var/cooldown = FALSE
 	var/tier = 1 //Upgrade via science
+	var/turf_type = /turf/open/space/basic
+
+//Hoc modo operatur. Do not question.
+/obj/machinery/computer/ship/mineral_magnet/stupidfuckingbabyaetherwhispmagnetvariantfortoproofdeckbecauseFUCKYOU
```
Comment (Jalleo) [inline nsv13/code/modules/mining/asteroid.dm:166]: A better rework of this is now required.

Comment (Jalleo) [reply nsv13/code/modules/mining/asteroid.dm:166]: Out of scope for this PR

**Review by Jalleo — DISMISSED** [review]
Things

File: nsv13/code/modules/mining/asteroid.dm (hunk 2 above)
Comment (Kmc2000) [reply nsv13/code/modules/mining/asteroid.dm:166]: Agreed this is a terrible fix but it works for now

**Comment by Kmc2000** [comment]
It's still sorta buggy, but we'll fix it on goal 2 of the roadmap

**Pull request merged**